src/uct/api/uct_def.h

```diff
@@ -398,7 +407,14 @@ typedef struct uct_cm_ep_server_conn_notify_args {
     uint64_t                   field_mask;
 
     /**
-     * Indicates the client's @ref ucs_status_t status.
+     * Indicates the client's @ref ucs_status_t status:
+     * UCS_OK                   - the client completed its connection
+     *                            establishment successfully.
+     * UCS_ERR_CONNECTION_RESET - the client's connection was reset during
+     *                            the connection establishment to the server.
+     * Other                    - an internal (server's) local error on the
```

Comment:
Otherwise - indicates an internal connection establishment error on the local (server) side